guide/src/main/asciidoc/api_specifications.adoc

```diff
@@ -49,8 +49,51 @@ According to usage, a conversion step to JSON might be necessary considering lim
 [rule, oas-tags]
 .Use tags to group operations
 ====
-When an API has many operations, use https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#tagObject[tags] to group them together.
-This will make the visual representation (SwaggerUI) more readable.
+When an API has many operations, https://spec.openapis.org/oas/v3.0.3#tag-object[tags] SHOULD be used to group them together.
+
+An operation SHOULD NOT have more than one tag.
+
+Each tag used on an operation SHOULD also be declared in the top level `tags` list of the OpenAPI document, with an optional description.
+
+The name of a tag SHOULD be stylized like a title, i.e. a word or space-separated word group starting with a capital letter.
+====
+
+Using tags will make the visual representation (e.g. SwaggerUI) more readable.
+Multiple tags on an operation duplicates the operation in SwaggerUI, which may cause confusion.
+
+Declaring each tag allows linting tooling to detect inconsistencies in tag names.
+
+.Using tags
+====
+```YAML
+tags:
+  - name: Employers
+    description: Managing employers
+  - name: Reference data
```

Comment:
Given https://github.com/belgif/rest-guide/issues/102#issuecomment-1908294570, I would add both styles as example, with Capitalized Words as the first. So e.g. add "Employment Details" above "Reference data".

Reply:
You probably wouldn't typically mix both naming styles within the same API though?

Reply:
Good point. In that case maybe just change the example from "Reference data" to "Reference Data"?

Reply:
why change? The other tag is "Employers" and the guideline doesn't prefer "Reference Data" above "Reference data".